invenio/modules/editor/templates/editor/index_base.html

```diff
@@ -0,0 +1,112 @@
+{#
+## This file is part of Invenio.
+## Copyright (C) 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{%- extends "page.html" -%}
+
+{% block global_bundles %}
+  {% bundles "editor.js", "invenio.css", "editor.css" %}
+{% endblock %}
+
+
+{%- block page_body -%}
+  <div class="container bibedit-container">
+    <div class="page-header">
+      <h2>Record Editor</h2>
+    </div>
+  {{ flashed_messages() }}
+  {%- block body %}
+    <div id="bibEditMenu">
+    <div class="bibEditMenuSection">
+    {% include "editor/recordmenu.html" %}
+    </div>
+    <div class="bibEditMenuSection">
+    {% include "editor/ticketsmenu.html" %}
+    </div>
+    <div class="bibEditMenuSection">
+    {% include "editor/fieldmenu.html" %}
+    </div>
+    <div class="bibEditMenuSection">
+    {% include "editor/viewmenu.html" %}
+    </div>
+    <div class="bibEditMenuSection">
+    {% include "editor/holdingpen.html" %}
+    </div>
+    <div class="bibEditMenuSection">
+    {% include "editor/undoredo.html" %}
+    </div>
+    <div class="bibEditMenuSection">
+    {% include "editor/history.html" %}
+    </div>
+    {% include "editor/circulation.html" %}
+    <div id="bibEditMenuSection">
+    {% include "editor/status.html" %}
+    </div>
+    <div class="bibEditMenuSection" align="right">
+    <a href="#" id="lnkSpecSymbols">Special symbols</a>
+    <a href="#" onclick='window.open(/help/admin/bibedit-admin-guide#2,width=640,height=600,left=150,top=150,resizable=yes,scrollbars=yes); return false;'>Help</a>
+    </div>
+    </div>
+    <div id='display_div'>
+      <strong>Display</strong> <br />
+      <ul id='focuson_list' class='list-plain'>
+        <li>
+          <input type="checkbox" name="references" id="focuson_references" value="references" checked/>
+          <label for="focuson_references">References</label>
+        </li>
+        <li>
+          <input type="checkbox" name="authors" id="focuson_authors" value="authors" checked/>
+          <label for="focuson_authors">Authors</label>
+        </li>
+        <li>
+          <input type="checkbox" name="others" id="focuson_others" value="others" checked/>
+          <label for="focuson_others">Others</label>
+        </li>
+          {% if config["CFG_INSPIRE_SITE"] %}
```

Comment:
Btw better solution would be to add `{% block focuson_list %}` and extend it in `inspire` overlay.

```jinja2
{% block focuson_list %}
{{ super() }}
<hr>
           <li>
             <input type="checkbox" name="curator" id="focuson_curator" value="curator"/>
             <label for="focuson_curator">Curator view</label>
           </li>
{% endblock %}
```